Tests/VaporToolboxTests/BuildTests.swift

```diff
@@ -0,0 +1,49 @@
+#if canImport(Testing)
+#if canImport(FoundationEssentials)
+import FoundationEssentials
+#else
+import Foundation
+#endif
+import Testing
+
+@testable import Build
+
+@Suite("Build Tests")
+struct BuildTests {
+    @Test("currentVersion")
+    func currentVersion() async throws {
+        let version = try await Build.currentVersion
+
+        let branchAndCommit = /^[a-zA-Z0-9._-]+ \([a-zA-Z0-9]{7}\)$/
+        // https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+        let semver =
+            /^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)(?:-(?P<prerelease>(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+(?P<buildmetadata>[0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
+
+        #expect(version.contains(branchAndCommit) || version.contains(semver))
+    }
+
+    @Test("withVersion", arguments: [true, false])
+    func withVersion(operationThrows: Bool) async throws {
```

Comment:
better test names perhaps? not immediately clear what they're trying to achieve.